client/ayon_core/pipeline/farm/pyblish_functions.py

```diff
@@ -754,10 +783,30 @@ def _create_instances_for_aov(instance, skeleton, aov_filter, additional_data,
     # go through AOVs in expected files
     for aov, files in expected_files[0].items():
         collected_files = _collect_expected_files_for_aov(files)
+        staging_dir = (
+            os.path.dirname(collected_files[0])
+            if isinstance(collected_files, (list, tuple))
+            else os.path.dirname(collected_files)
+        )
+
+        if (
+            frames_to_render is not None
+            and isinstance(collected_files, (list, tuple))  # not single file
+        ):
+            frames_to_render = get_real_frames_to_render(frames_to_render)
+            collections, _ = clique.assemble(collected_files)
+            collected_files = _get_real_files_to_render(
+                collections[0], frames_to_render)
+        else:
+            frame_start = int(skeleton.get("frameStartHandle"))
+            frame_end = int(skeleton.get("frameEndHandle"))
+            frames_to_render = list(range(frame_start, frame_end + 1))
 
-        expected_filepath = collected_files
         if isinstance(collected_files, (list, tuple)):
-            expected_filepath = collected_files[0]
+            expected_filepath = os.path.join(staging_dir, collected_files[0])
```

Comment:
How did it before worked as filepaths, and not it is filename? Isn't that because `_collect_expected_files_for_aov` returns filepaths, but `_get_real_files_to_render` does return filenames? If yes, then we should change it to filepaths...